### ISSUE TITLE
I seem to be getting a session error with Facebook.

### ISSUE THREAD

**Issue by Zombie-Ryushu** [DMJC/purple-facebook#4]
Error validating access token: The session has been invalidated because the user changed their password or Facebook has changed the session for security reasons. (190)

**Comment by Zombie-Ryushu** [comment]
Facebook is now telling me my Username and Password is incorrect no matter what.

**Comment by krakenkall** [comment]
> Facebook is now telling me my Username and Password is incorrect no matter what.

logout and login browser, remove facebook account in pidgin then add it back, its work now bc some sessions is expired.

**Comment by Zombie-Ryushu** [comment]
Okay, I got it working!

**Comment by lukasz-r** [comment]
I've just removed the old plugin from my distro's official repo (OpenSUSE Tumbleweed) and installed the new plugin using this repo. Unfortunately I can't login, I keep getting the

Invalid username or password (401)

error.

I tried changing the Facebook password, logging in and out to Facebook account on the website and removing and adding the account in Pidgin, so far no success.

> Okay, I got it working!

Do you remember steps leading to your success?

**Comment by Zombie-Ryushu** [comment]
Yes, I had to change my password, and then log out of the account entirely on everything. Then, DELETE the account from the accounts in Pidgin, re-login, and then re-add the Pidgin account.

Pidgin and Spectrum 2 will hold on to a Token that is only visible if you open accounts.xml, this Token changes and if it is still present Facebook will invalidate the login, I think.

**Comment by Metalhead33** [comment]
> Yes, I had to change my password, and then log out of the account entirely on everything. Then, DELETE the account from the accounts in Pidgin, re-login, and then re-add the Pidgin account.

I tried doing that, didn't work.
